mappings/cvss_v3.json

```diff
@@ -0,0 +1,718 @@
+{
+  "metadata": {
+    "default": "AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:N"
+  },
```

Comment:
what do you think about putting the version in the metadata instead of the filename?

Reply:
But what happens if we ever decide to add cvss v2 or a theoretical cvss v4? Having a file `cvss.json` with `metadata.{cvss_}version = 3.0` would make that harder. Not that I think those are likely anytime soon but I'd rather not block it if it's easily avoided.

Reply:
I could be misunderstanding the situation, but I was thinking of treating it the same way we do with the vrt json.  Since no single version of vrt will have mappings to multiple versions of cvss, when a new cvss version comes out we would need to create a new version of the vrt to include the updated mapping.  All nodes from previous vrt versions will always map to the latest cvss version through the deprecated node mapping.

@barnett or @plr0man may have more insight on the intended use?

Reply:
another idea could be to only have the `latest_version` in the metadata and mappings to different versions as keys in the json like:

```json
{
   "id": "server_security_misconfiguration",
   "children": [
     {
       "id": "unsafe_cross_origin_resource_sharing",
       "cvss_v3": "AV:N/AC:L/PR:N/UI:R/S:U/C:H/I:L/A:N",
       "cvss_v4": "AV:N/AC:L/PR:N/UI:R/S:U/C:H/I:L/A:N"
     }
   ]
}

Reply:
I think the default CVSS version is more a matter of customers' preference. If CVSS v4 ever comes out we would still be using CVSS v3 by default, until we would see most of our customers switch over to v4, which might or might not happen. If we had customers that prefer v2, we might consider adding that mapping too.

Reply:
Ah ok.  I didn't realize that we'd be maintaining multiple cvss mappings simultaneously.  Good to know.